tests/pyaml_env_tests/test_base_config.py

```diff
@@ -1,6 +1,6 @@
 import os
 import unittest
-from pyaml_env import BaseConfig
+from src.pyaml_env import BaseConfig
```

Comment:
Is the `src.` necessary for your env? It doesn't seem right since `pyaml_env` should be installed. Can you please check? Same below :)